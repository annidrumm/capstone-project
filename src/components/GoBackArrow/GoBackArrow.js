import React from 'react'
import backIcon from '../../icons/back.svg'
import styled from 'styled-components/macro'
import { useHistory } from 'react-router-dom'

export default function GoBackArrow() {
  const history = useHistory()
  const routeChange = () => {
    let path = `/`
    history.push(path)
  }
  return <GoBackStyled src={backIcon} alt="arrow left" onClick={routeChange} />
}

const GoBackStyled = styled.img`
  position: absolute;
  left: 30px;
  margin-top: 10px;
  cursor: pointer;
`
