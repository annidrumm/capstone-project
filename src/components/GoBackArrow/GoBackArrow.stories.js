import React from 'react'
import backIcon from '../../icons/back.svg'
import GoBackArrow from './GoBackArrow'

export default {
  title: 'Components/Button',
  component: GoBackArrow,
}

export const goBack = () => {
  return <img src={backIcon} alt="arrow left" />
}
