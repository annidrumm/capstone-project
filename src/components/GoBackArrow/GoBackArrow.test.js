import React from 'react'
import renderer from 'react-test-renderer'
import GoBackArrow from './GoBackArrow'
import '@testing-library/jest-dom/extend-expect'

describe('GoBackArrow', () => {
  it('renders GoBackArrow correctly', () => {
    const tree = renderer.create(<GoBackArrow />)
    expect(tree).toMatchSnapshot()
  })
})
