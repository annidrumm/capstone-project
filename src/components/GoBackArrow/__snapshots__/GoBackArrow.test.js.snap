// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoBackArrow renders GoBackArrow correctly 1`] = `
<img
  alt="arrow left"
  className="GoBackArrow__GoBackStyled-m93ye8-0 kcjJZM"
  id="goBack"
  onClick={[Function]}
  src="back.svg"
  value="goBack"
/>
`;
